### PR TITLE
Add some basic pinning tests

### PR DESCRIPTION
**Pull request by mjbvz**
Add some basic tests around the library's existing behavior

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    assert.equal(
+      md.render([
+        '----',
+        'x: 1',
+        '---',
+        '# Head'
+      ].join('\n')),
+      '');
+
+    assert.equal(foundFrontmatter, 'x: 1\n---');
```
Comment (craigdmckenna) [inline test/index.js:95]: Should this assert `undefined`? Or, are you saying that it's not currently working as intended? What I mean is that if the fences are not equal length, then it should not be considered front matter.

**Review by craigdmckenna — COMMENTED** [review]
Thank you for your contribution, @mjbvz. Please see my comment.

File: test/index.js (hunk 1 above)
Comment (mjbvz) [reply test/index.js:95]: Yes I think it is not be behaving as expected here. I confirmed that `foundFrontmatter` is populated in this case

Comment (craigdmckenna) [reply test/index.js:95]: Ok. We should probably change this test case, then. It's documented that way.

> Opening and closing fences must be the same number of dash characters

...then see what we can do to fix it.

Also, I'd like to setup a build process, possibly before merging this.

Thoughts?

Comment (mjbvz) [reply test/index.js:95]: @craigdmckenna The intent of these tests is just to pin the existing behavior. Perhaps worth tackling any identified errors + setting up build in separate changes?

**Comment by mjbvz** [comment]
Anything blocking this?

File: test/index.js (hunk 1 above)
Comment (ghost) [reply test/index.js:95]: This is just the behavior.
Of course we need to decide to consider as a bug or not about this behavior.
If you have other ideas, please leave an issue!

**Review by ghost — APPROVED** [review]
LGTM
Thank you for your contribution!

**Pull request merged**